kingfisher_scrapy/pipelines.py

```diff
@@ -40,3 +40,58 @@ def process_item(self, item, spider):
             self.files.add(key)
 
         return item
+
+
+class LatestReleaseDate:
+    def __init__(self):
+        self.processed = set()
+
+    def process_item(self, item, spider):
+        if spider.latest and (isinstance(item, FileItem) or isinstance(item, File)):
```

Comment:
We can use this syntax instead:

```suggestion
        if spider.latest and isinstance(item, (File, FileItem)):
```